upgrade/1.5.2/README.md

```diff
@@ -0,0 +1,306 @@
+bullets together if it makes sense (eg. many Paradise bullets were combined into a single "Support
+for Paradise hardware" bullet..
+
+* Added support for Paradise hardware
+* Fixed issue in `conman` in order to support Paradise hardware
+* Fixed issue in `run_hms_ct_tests.sh` which caused false positives for CDU switches
+* Enhanced `hmcollector` logging
+* FAS now waits for the time limit to expire when verifying update. FASUpdate.py script also updated to accept a --timeLimit parameter to change the preset time limit
+* Required environment variables now set to address breaking changes in the latest ca-certificates RPM
+* Update IMS recipe builds to use new DST signing key
+* Mitchell Harding: CASMCMS-8971
+* Mitchell Harding: CASMCMS-8973
+* Mitchell Harding: CASMCMS-8978
+* Mitchell Harding: CASMCMS-8186
+* Mitchell Harding: CASMCMS-8438
+* Mitchell Harding: CASMINST-6861
+* Mitchell Harding: CASMTRIAGE-6993
+* Mitchell Harding: CASMTRIAGE-7019
+* Mitchell Harding: CASMHMS-5838
+* Mitchell Harding: CASMCMS-9001
+* Mitchell Harding: CASMCMS-8997
+* Mitchell Harding: CASMCMS-8998
+* Mitchell Harding: CASMCMS-9017
+* Mitchell Harding: CASMCMS-9018
+* Mitchell Harding: CASMCMS-9015
+* Mitchell Harding: CASMCMS-9022
+* Russell Bunch: CASMTRIAGE-6991
+* Russell Bunch: MTL-2393
+* Mikhail Tupitsyn: CASMPET-7093
+* Chris Spiller: CASMNET-2221 (to be replaced by Chris as part of CASMNET-2179)
+
+## Limitations
+
+* Power capping support for Paradise Hardware
+  * Power caps are applied to the processor complex, not to the entire node
+  * A power cap cannot be applied when node power is off
+  * If a power cap is set when node power is on, it will be removed if the node is ever powered off.  A new power cap must be applied after the node is powered back on
+
+## Steps
+
+1. [Preparation](#preparation)
+1. [Setup Nexus](#setup-nexus)
+1. [Upgrade services](#upgrade-services)
+1. [Upload NCN images](#upload-ncn-images)
+1. [Update management node CFS configuration](#update-management-node-cfs-configuration)
+1. [Update test suite packages](#update-test-suite-packages)
+1. [Verification](#verification)
+1. [Take Etcd manual backup](#take-etcd-manual-backup)
+1. [NCN reboot](#ncn-reboot)
+1. [Complete upgrade](#complete-upgrade)
+
+### Preparation
+
+1. Validate CSM health.
+
+   See [Validate CSM Health](../../operations/validate_csm_health.md).
+
+   Run the CSM health checks to ensure that everything is working properly before the upgrade starts.
+   After the upgrade is completed, another health check is performed.
+   It is important to know if any problems observed at that time existed prior to the upgrade.
+
+   IMPORTANT: See the CSM Install Validation and Health Checks procedures in the documentation for the CURRENT CSM version on the system.
+   The validation procedures in the CSM documentation are only intended to work with that specific version of CSM.
```

Comment:
This caveat is really only needed when upgrading from a different minor version. The docs for CSM 1.5.1 are the same as for 1.5.2, so this does not apply here.

```suggestion
```

Reply:
Removed.